conans/client/command.py

```diff
@@ -330,7 +330,7 @@ def info(self, *args):
         parser.add_argument("--graph", "-g", action=OnceArgument,
                             help='Creates file with project dependencies graph. It will generate '
                             'a DOT or HTML file depending on the filename extension')
-        parser.add_argument("--install-folder", "-if", action=OnceArgument,
+        parser.add_argument("--install-folder", "--install_folder", "-if", action=OnceArgument,
```

Comment:
Duplicated argument.

Reply:
It is just adding ``install_folder name``, as done with ``conan build --install-folder``, ``conan build -install_folder``, ``conan build -if``

Reply:
Ah! Sorry, I missed the _ underscore

Do not add them. They were in the other commands for backwards compatibility. New commands arguments should be all with hyphen ``-``

Reply:
So in that case this PR does not make sense.

It will then just add the functionality to print complete option names in error messages and it is not a clean implementation.

Reply:
Actually, I would say we might consider removing the old _ underscore arguments for 1.0!